### PR TITLE
Add E2E tests for preferential optimization

### PR DESCRIPTION
**Pull request by HideakiImamura**
## Contributor License Agreement

This repository (``optuna-dashboard``) and [Goptuna](https://github.com/c-bata/goptuna) share common code.
This pull request may therefore be ported to Goptuna.
Make sure that you understand the consequences concerning licenses and check the box below if you accept the term before creating this pull request.

*  [x] I agree this patch may be ported to [Goptuna](https://github.com/c-bata/goptuna) by other Goptuna contributors.

## Reference Issues/PRs

Following https://github.com/optuna/optuna-dashboard/pull/484, I add e2e tests using playwright for preferential optimization.

## What does this implement/fix? Explain your changes.

This PR adds the e2e test file `test_preferential_optimization.py`. This tests the followings.
- Accessing the trial list page and confirm that the trial list page is displayed. Check that all trials are running. 
- The first trial are marks as completed with "Bad" feedback. Check that it is completed and the best trial.
- The second trial are marks as completed with "So-so" feedback. Check that it is completed and the best trial.
- The third trial are marks as completed with "Good" feedback. Check that it is completed and the best trial.
- The fourth trial are marks as failed. Check that it is failed.

The following screenshot was generated by the `playwright --screenshot on` option.
test-finished-1

### CODE REVIEW

**Comment by HideakiImamura** [comment]
@keisuke-umezawa Could you review this PR?

**Comment by keisuke-umezawa** [comment]
@HideakiImamura Seems not to work with optuna==2.10.0. Could you check the code?

**Comment by HideakiImamura** [comment]
`optuna==1.10.0` does not support the preferential optimization, so the error occurs. I think we should skip the tests.

**Comment by keisuke-umezawa** [comment]
@HideakiImamura 
Thank you for the explanation. I understand. I will check this PR.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -83,6 +83,7 @@ $ pytest python_tests/
 
 ```
 $ pip install -r requirements.txt
+$ playwright install
```
Comment (keisuke-umezawa) [inline CONTRIBUTING.md:86]: thx!

**Review by keisuke-umezawa — APPROVED** [review]
Great example of e2e test! Thank you.

**Pull request merged**